implementations.yaml

```diff
@@ -84,7 +105,10 @@ kems:
       - name: saber
         implementation: clean
```

Comment:
SABER now has AVX2 implementations, which could be made available

Reply:
AVX2 implementation added and tested. Working great. Note added to changelog.